patterns/rightsizing-virtual-machines.md

```diff
@@ -0,0 +1,45 @@
+# Rightsizing oversized VMs
+
+## Version
+1.0
+
+## Submitted By
+Bill Johnson
+
+## Published Date
+TBD
+
+## Intent
+
+Choose servers that are the right size for the task at hand.
+
+## Tags
+Cloud, Compute, Cloud Engineer, Small
```

Comment:
Turns out with docusaurus if we add tags to the front matter at the top of the markdown as a list, I think there is a way to render the catalogs automatically.

Something like

```
---
tags:
 - cloud
 - compute
 - role-cloud-engineer
 - size-small
---
```